homeassistant/components/rituals_perfume_genie/strings.json

```diff
@@ -17,5 +17,36 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "charging": {
+        "name": "[%key:component::binary_sensor::entity_component::battery_charging::name%]"
+      }
+    },
+    "number": {
+      "perfume_amount": {
+        "name": "Perfume amount"
+      }
+    },
+    "select": {
+      "room_size_square_meter": {
+        "name": "Room size"
+      }
+    },
+    "sensor": {
+      "battery_percentage": {
+        "name": "[%key:component::sensor::entity_component::battery::name%]"
+      },
+      "fill": {
+        "name": "Fill"
+      },
+      "perfume": {
+        "name": "Perfume"
+      },
+      "wifi_percentage": {
+        "name": "Wi-Fi signal"
```

Comment:
Only thing I dislike is the Wi-Fi spelling. Although they are used both throughout HA. Wi-fi finds 23 files using it in components, wifi finds 54+ files.

Reply:
It is following the Microsoft style guides for writing, which we have adopted. It specifically marks the use of "Wi-Fi", as that is the official registered mark for it.

Reply:
Right, I can live with Wi-Fi, only the annoying thing is that the entity name is now wi_fi, and that looks bad imo

Reply:
👍 I'll make a note of that and fix it in an next iteration. We can just suggest an object ID for that.